README.md

```diff
@@ -33,31 +34,35 @@ Our mission is our singular purpose for existing. All of our work should contrib
 
 # Vision: what we hope to achieve
 
-Our vision is the future state we hope to create in pursuit of our mission: 
+Our vision is the future state we hope to create in pursuit of our mission:
 
 **Able Health will be the source of truth for quality-based payments to all US physicians.**
 
 # Values: who we are
 
 Our values are what we believe to be true and how we conduct ourselves at all times, internally and externally. They are the qualities that define an Abler, to be used as a moral compass and screening for new team members. We are...
 
-## Curious experts 
+## Curious experts
 
 You hungrily gather new knowledge. You honor a diversity of strengths and perspectives. You welcome respectful debate. You make space for everyone to get their ideas on the table. You listen with an open mind. You bring all available information to bear on your decisions. You are a mentor and a mentee for life.
 
 ## Comrades in service
 
-You want to improve the lives of others, whether those people are customers, coworkers, or strangers. You recognize that small interpersonal actions build lifelong relationships. You respect that everyone has deeply personal motivators. You want to lift up those around you rather than promote yourself. You seek to spread delight. 
+You want to improve the lives of others, whether those people are customers, coworkers, or strangers. You recognize that small interpersonal actions build lifelong relationships. You respect that everyone has deeply personal motivators. You want to lift up those around you rather than promote yourself. You seek to spread delight.
 
-## Biased toward action 
+## Biased toward action
 
 You act on problems when you see them. You work in service of outcomes, not appearances. You deliver incrementally to get to the right solution faster. You follow through on your promises. You produce high-quality work. You speak up when you disagree. You communicate directly. You own your failures, and you learn from them.
 
+## Generosity of spirit
+
+ You invest in others, not only tasks. You have patience and perspective to lift the skills and expertise of those around you, because you know there only so much an individual can accomplish on their own. You spend time to make sure that more junior or newer Able team members not only understand what they are doing, but also **why** they are doing it.
+
```

Comment:
I really like that! 

Emily Richmond, MPH
emily@ablehealth.com
ablehealth.com

> On May 27, 2016, at 12:42 AM, Sean Knox <notifications@github.com> wrote:
> 
> In README.md:
> 
> >  
> >  You act on problems when you see them. You work in service of outcomes, not appearances. You deliver incrementally to get to the right solution faster. You follow through on your promises. You produce high-quality work. You speak up when you disagree. You communicate directly. You own your failures, and you learn from them.
> >  
> > +## Generosity of spirit
> > +
> > + You invest in others, not only tasks. You have patience and perspective to lift the skills and expertise of those around you, because you know there only so much an individual can accomplish on their own. You spend time to make sure that more junior or newer Able team members not only understand what they are doing, but also **why** they are doing it.
> > +
> This is the only change—looks like my text editor auto-removed some whitespace.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>